sdk/extensions/chronopost/karrio/providers/chronopost/tracking.py

```diff
@@ -0,0 +1,84 @@
+import typing
+from urllib import response
+import karrio.lib as lib
+import karrio.core.models as models
+import karrio.providers.chronopost.utils as provider_utils
+import karrio.providers.chronopost.error as provider_error
+import chronopost_lib.trackingservice as chronopost
+
+
+def parse_tracking_response(
+    responses: typing.List[lib.Element],
```

Comment:
since responses have been bundled together, you will get a single parent element for all

```suggestion
    responses: lib.Element,
```